src/components/VButton.vue

```diff
@@ -143,6 +149,13 @@ const VButton = defineComponent({
     const typeRef = ref(propsRef.type.value)
     const supportsDisabledAttributeRef = ref(true)
 
+    const isActive = computed(() => {
+      return (
+        propsRef.pressed.value ||
+        attrs['aria-pressed'] ||
+        attrs['aria-expanded']
```

Comment:
These _might_ need to be compared against the string `"true"` value as `"false"` will be truthy in these cases... But don't quote me on that! I feel like Vue does some magical stuff with these attributes to cast a boolean literal to a string or vice-versa. Just not sure when exactly that happens 😅

Reply:
I will test more tomorrow!

Reply:
I tried logging the values, and they are boolean (or undefined)